.github/workflows/examples-linux-arm.yaml

```diff
@@ -67,6 +67,7 @@ jobs:
                         --target linux-arm64-chip-tool-ipv6only \
                         --target linux-arm64-minmdns \
                         --target linux-arm64-thermostat-no-ble \
+                        --target linux-arm64-lock \
```

Comment:
do we need to validate the arm cross compile? Trying to minimize the amount of CI time - we do not build light for example, just want to have a subset here.

Unless there is reason to believe the arm64 build will show things that the x64 will not (and nor will things like all-clusters) please remove. The other builds try to mix variants (all clusters is everything, ipv6only and no-ble variants are split, minmdns validates dns clients and servers)

Reply:
Removed!